cluster-autoscaler/core/scaledown/actuation/drain.go

```diff
@@ -245,11 +243,7 @@ func evictPod(ctx *acontext.AutoscalingContext, podToEvict *apiv1.Pod, isDaemonS
 	return status.PodEvictionResult{Pod: podToEvict, TimedOut: true, Err: fmt.Errorf("failed to evict pod %s/%s within allowed timeout (last error: %v)", podToEvict.Namespace, podToEvict.Name, lastError)}
 }
 
-func podsToEvict(ctx *acontext.AutoscalingContext, nodeName string) (dsPods, nonDsPods []*apiv1.Pod, err error) {
-	nodeInfo, err := ctx.ClusterSnapshot.NodeInfos().Get(nodeName)
-	if err != nil {
-		return nil, nil, err
-	}
+func podsToEvict(ctx *acontext.AutoscalingContext, nodeInfo *framework.NodeInfo) (dsPods, nonDsPods []*apiv1.Pod, err error) {
```

Comment:
Looks like this cannot really return an error anymore, can you update the function signature?

Reply:
Fixed.